libsolidity/ast/ExperimentalFeatures.h

```diff
@@ -35,10 +36,11 @@ enum class ExperimentalFeature
 	TestOnlyAnalysis
 };
 
-static std::map<ExperimentalFeature, bool> const ExperimentalFeatureOnlyAnalysis =
+static std::set<ExperimentalFeature> const ExperimentalFeatureWithoutWarning =
 {
-	{ ExperimentalFeature::SMTChecker, true },
-	{ ExperimentalFeature::TestOnlyAnalysis, true },
+	ExperimentalFeature::ABIEncoderV2,
+	ExperimentalFeature::SMTChecker,
```

Comment:
Sure about this one? ;)

Reply:
Yeah, is this meant to out of experimental with warning as well?

Reply:
No, it has always been....
Am I missing something here?

Reply:
@chriseth you're removing the experimental warning for the SMTChecker as well. Was that intended?

Reply:
SMTChecker was inside this list before the change.